utils/_context/_scenarios.py

```diff
@@ -858,6 +913,9 @@ class scenarios:
         additional_trace_header_tags=("header-tag1:custom.header-tag1", "header-tag2:custom.header-tag2"),
     )
 
+    onboarding_host = OnBoardingScenario("ONBOARDING_HOST")
```

Comment:
Could you add a small comment on whereto look for scenario details (link to yml file)